python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -87,7 +89,8 @@ class DiracBase(IBackend):
                                 doc='DIRAC API commands to add the job definition script. Only edit '
                                 'if you *really* know what you are doing'),
         'settings': SimpleItem(defvalue={'CPUTime': 2 * 86400},
-                               doc='Settings for DIRAC job (e.g. CPUTime, BannedSites, etc.)')
+                               doc='Settings for DIRAC job (e.g. CPUTime, BannedSites, etc.)'),
+        'credential_requirements': ComponentItem('CredentialRequirement', defvalue=DiracProxy()),
```

Comment:
If they need to change anything then they are indeed required to know how to create one, yes:
```python
j = Job()
j.backend = Dirac(credential_requirements=DiracProxy(group='my_group'))
```